tests/file_storage/tests.py

```diff
@@ -403,6 +403,19 @@ def test_delete_no_name(self):
         with self.assertRaises(AssertionError):
             self.storage.delete('')
 
+    def test_seekable(self):
```

Comment:
Can we put this test in `tests/files`? I don't think it's particular to file storage. Also, it seems we should check the result of `file.seekable()`, not just verify the attribute exists. I'd just skip the test on Python 2.